EosPayload/drivers/RadioDriver.py

```diff
@@ -66,14 +66,6 @@ def setup(self) -> None:
                 self._logger.error(f"radio port not open: {e}")
                 time.sleep(10)
 
-    @staticmethod
-    def get_device_id() -> Device:
-        return Device.RADIO
-
-    @staticmethod
-    def get_device_name() -> str:
-        return "radio-driver"
-
     @staticmethod
     def read_thread_enabled() -> bool:
```

Comment:
The config is meant to be for the end user to configure the behavior of the driver, and I can't think of a case where changing which threads are enabled wouldn't immediately break things.

Another solution would be putting thread_enabled functions in DriverBase that use the config, so that it's up to the programmer if the threads can be configured or not. Not sure if you think that's worth it or not.